compiler/modules/front/parseStmt.bal

```diff
@@ -158,7 +158,15 @@ function finishAssignStmt(Tokenizer tok, LExpr lValue) returns AssignStmt|err:Sy
 function parseCompoundAssignStmt(Tokenizer tok, LExpr lValue, CompoundAssignOp op) returns CompoundAssignStmt|err:Syntax {
     check tok.advance();
     Expr rexpr = check parseExpr(tok);
-    CompoundAssignStmt stmt = { lValue, rexpr , op, pos: tok.currentPos()};
+    string opStr = op.toString();
```

Comment:
Why toString?

Reply:
The idea was to remove the `=` by getting a substring without last character

Reply:
It is a string already. You may be running into a jBallerina bug... If so, add a cast and put a JBUG comment.